TypewiseAlert/AlertTargets/EmailAsTarget.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using static TypewiseAlert.TypewiseAlert;
+using TypewiseAlert.Interfaces;
+
+namespace TypewiseAlert.AlertTargets
+{
+  public class EmailAsTarget: IBreachAlert
+  {
+    static Dictionary<BreachType, IEmailSent> breachAlerters = new Dictionary<BreachType, IEmailSent>
+      {
+
+            {BreachType.TOO_LOW, new EmailLowTemperature() },
+            {BreachType.TOO_HIGH, new EmailHighTemperature() },
+            {BreachType.NORMAL, new EmailLowTemperature() },

```

Comment:
The IEmailSent Object needs to be EmailNormalTemperature. This seems to be a bug. Ideally there should have been a test case that could have caught this error.

Looks like test case to cover this aspect is missing. Please add test case, ensure that it fails first and then correct the production code.

Reply:
Corrected👍